### ISSUE TITLE
Change needed in the programme "policy.py" and 'current_law_policy.json" to consider inflation effect

### ISSUE THREAD

**Issue by shekharkhetan** [TPRU-demo/pitaxcalc-demo#57]
Inspite of certain variables being true for cpi_inflatable  , the programme was giving the following bug:

`````````````````
Traceback (most recent call last):
  File "app14.py", line 17, in <module>
    reform = Calculator.read_json_param_objects('app14_reform.json', None)
  File "C:\Users\hp\Desktop\wb training-sept18\pitaxcalc-demo\taxcalc\calculator.py", line 731, in read_json_param_objects
    rpol_dict = Calculator._read_json_policy_reform_text(txt)
  File "C:\Users\hp\Desktop\wb training-sept18\pitaxcalc-demo\taxcalc\calculator.py", line 951, in _read_json_policy_reform_text
    prdict = Calculator._convert_parameter_dict(tdict)
  File "C:\Users\hp\Desktop\wb training-sept18\pitaxcalc-demo\taxcalc\calculator.py", line 1054, in _convert_parameter_dict
    year = int(skey)
ValueError: invalid literal for int() with base 10: 'cpi_inflated'
`````````````````````````````````````

**Comment by martinholmer** [comment]
@shekharkhetan, Thanks for your bug report in issue #57 concerning the simulator crash when specifying a reform that turns on CPI-indexing of a policy parameter.  After proposing changes that fix bug reports #55 and #56, I plan to investigate the bug you reported in issue #57.  I'm going to start doing that investigation by seeing if that bug exists in our USA Tax-Calculator.  If it does, I'll fix it there and then also in the `TPRU-demo/pitaxcalc-demo` repository.  If the bug does not exist in the USA Tax-Calculator, then I'll fix it just in the `TPRU-demo/pitaxcalc-demo` repository.  There is also another possibility: I've forgotten how to write a JSON reform file that turns on CPI-indexing of a policy parameter.  We discovered on Friday in New Delhi that the USA Tax-Calculator documentation and test suite are not as good as they should be in showing how to specify this kind of reform.  I hope fixing #57 is that easy, but more likely there is a flaw in the code.  I'll be reporting here what I learn in the investigation over the course of the first week in October.

@indira-tpru @adc2-tpru @keshavchoudhary87 @sebastiansajie

**Comment by martinholmer** [comment]
See pull request #61 for a proposed resolution for issue #57.

@indira-tpru @adc2-tpru @keshavchoudhary87 @sebastiansajie

**Comment by martinholmer** [comment]
The merge of pull request #61 resolves issue #57, which is now being closed.
